projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid.component.html

```diff
@@ -58,15 +60,14 @@
         | treeGridFlattening:id:expansionDepth:expansionStates:pipeTrigger
         | treeGridPaging:page:perPage:id:pipeTrigger
         | treeGridSummary:hasSummarizedColumns:summaryCalculationMode:summaryPosition:id:pipeTrigger:summaryPipeTrigger"
-
         let-rowIndex="index" [igxForScrollOrientation]="'vertical'"  [igxForScrollContainer]='verticalScroll'
         [igxForContainerSize]='calcHeight' [igxForItemSize]="rowHeight" #verticalScrollContainer (onChunkPreload)="dataLoading($event)">
             <ng-template #record_template>
                 <igx-tree-grid-row [gridID]="id" [index]="rowIndex" [treeRow]="rowData" #row>
                 </igx-tree-grid-row>
             </ng-template>
             <ng-template #summary_template>
-                <igx-grid-summary-row [gridID]="id" [summaries]="rowData.summaries" [firstCellIndentation]="rowData.cellIndentation" [index]="rowIndex" class="igx-grid__summaries--body" #summaryRow>
+                <igx-grid-summary-row [gridID]="id" #row [summaries]="rowData.summaries" [firstCellIndentation]="rowData.cellIndentation" [index]="rowIndex" class="igx-grid__summaries--body" #summaryRow>
```

Comment:
So, this is both `#row` and `#summaryRow`?

Reply:
This is probably an incorrect merge issue. I'll remove it.